get-booking/lambda.js

```diff
@@ -22,12 +22,12 @@ exports.handler = (event, context, callback) => {
 	logger.debug('Received event: ', JSON.stringify(event));
 
 	// invoke business logic
-	const { DrivingLicenceNumber, Date } = event;
-	const getBookingService = new GetBookingService();
-	getBookingService.drivingLicenceNumber = DrivingLicenceNumber;
-	getBookingService.date = Date;
+	const { Request } = event;
+	const { DrivingLicenceNumber, Date } = Request;
 
-	// return success
-	exit(callback, null, getBookingService().getBooking());
 
+	// return success
+	GetBookingService.getBooking(DrivingLicenceNumber, Date, (response) => {
```

Comment:
This is confusing - it superficially looks like a 'then' function for a Promise, but it isn't.
If the service function ('getBooking') is asynchronous, it should return a Promise.

Reply:
Done